docs/configuration/translation-strings.md

```diff
@@ -0,0 +1,65 @@
+
+[[toc]]
+
+## How It Works
+
+<video autoplay muted loop controls>
+<source src="https://cdn.directus.io/docs/v9/configuration/translation-strings/translation-strings-20220615A/translation-strings-20220615A.mp4" type="video/mp4" />
+</video>
+
+When creating a Field, you may want to add a note or other description along with it to clarify its context or purpose.
+While you always have the option to type in a static value, Translation Strings let you group together translations
+which display dynamically, whenever the app language is changed. There are two locations where the language could be
+set. First, administrators can set the Project's default language under **Settings > Project Settings**. Second, Users
+can choose their own personal [language preference](/app/user-directory/#user-preferences) from the user details page.
+
+## Create and Manage a Translation String
+
+<video autoplay muted loop controls>
+<source src="https://cdn.directus.io/docs/v9/configuration/translation-strings/translation-strings-20220616A/create-and-manage-a-trasnlation-string-20220616A.mp4" type="video/mp4" />
+</video>
+
+To create and manage a Translation String, follow these steps.
+
+1. Navigate to **Settings > Translation Strings**.
+2. Click <span mi btn>add</span> in the page header and a drawer will open.
+3. Add a key and click **"Create New"** and another drawer will open.
+4. Select the language and type in the corresponding translation.
+5. Click <span mi btn>check</span> to confirm and add the translation.
+6. Repeat steps 3-5 as desired.
+7. Optional: Click <span mi icon>close</span> to delete a translation.
+8. Click <span mi btn>check</span> to confirm and save the Translation Strings.
+
+## Assign and Remove a Translation String
+
+<video autoplay muted loop controls>
+<source src="https://cdn.directus.io/docs/v9/configuration/translation-strings/translation-strings-20220616A/assign-and-remove-translation-string-20220616A.mp4" type="video/mp4" />
+</video>
+
+Throughout the **Data Model** as well as **Presets and Bookmarks** sections in the Settings Module, you will notice
+certain input fields have a <span mi icon>translate</span> icon on them. While can always type static text into these
+input fields, this <span mi icon>translate</span> icon indicates that you have the option assign a Translation String.
+To assign a Translation String, follow these steps.
+
+1. Navigate to the input that you'd like to add a Translation String to.
+2. There are two ways to assign a Translation String:
+   - Click <span mi icon>translate</span> and a dropdown menu with all Translation Strings will appear.
+   - Type `$t:translation-string-key` and hit enter.
+3. Choose a Translation String key as desired.
+4. Optional: Click to remove the Translation String.
+5. Click <span mi btn>check</span> to confirm.
+
+Now your Translation String is set! Whenever the app's language is set, the corresponding translation string will be
```

Comment:
> Whenever the app's language is set,

Confusing. It doesn't have to do with when the language set... just what language is being used. It would be clearer to say "Now the language-appropriate text will be shown based on the **current language of the app**. Again, the bold would link to the "Changing the Apps Language" section.

Reply:
ok.